File: 1 OOP course Uber/Python/main.py
from car     import Car
from account import Account
from uberBlack import UberBlack

if __name__ == "__main__":
    print("Hola mundo")
    
    
    car = Car("MGE277",  Account("Andrés Cueto" , "Visa") )
    car.dataCar()
    
    car2 = Car("NPR069", Account("Alejandro Velez" , "INE") )
    car2.dataCar()
    
    car3 = UberBlack("NPM5576", Account("Scarlette Bello", "kjno"), ["hola"], ["adios"] )
    car3.dataCar()

File: 1 OOP course Uber/Java/main.java
class Main{

    public static void main(String[] args) {
        
        UberX car = new UberX("AMQ123", new Account("Andres Cueto", "AND123"), "Suzuki", "Swift");
        car.license = "AMQ123";
        car.setPassenger(6);
        car.printDataCar();

        // Car car2 = new Car("YMI270", new Account("Scarlette Bello", "SCR123"));
        // car2.printDataCar();
    }

}